code/modules/clothing/glasses/glasses.dm

```diff
@@ -119,6 +119,14 @@
 	see_in_dark = 8
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 
+/obj/item/clothing/glasses/meson/sunglasses
+	name = "meson HUD sunglasses"
+	desc = "Sunglasses with an inbuilt scanner that can see through both walls and flooring.."
+	icon_state = "sunhudmeson"
+	see_in_dark = 1
```

Comment:
sunglasses giving a low light buff is a little odd tbh.
unless im misinterpreting this

Reply:
Just a leftover from copypasting janihud sunglasses over for the format, I assume that was on it to made the janihud itself look better. I'll toss it.